apps/condo/index.js

```diff
@@ -174,14 +174,17 @@ class FeaturesMiddleware {
 
 class PaymentLinkMiddleware {
     async prepareMiddleware () {
-        const app = express()
-        const router = new PaymentLinkRouter()
+        const features = await featureToggleManager.fetchFeatures()
```

Comment:
We have common way for checking feature flag state, please use it https://github.com/open-condo-software/condo/pull/2450/files

Reply:
ok